docs/app/pages/components/components-sections/tables/selection/selection.component.html

```diff
@@ -151,6 +151,9 @@
     <tr uxd-api-property name="selected" type="boolean">
         Defines whether or not this item is currently selected.
     </tr>
+    <tr uxd-api-property name="preventAriaSelected" type="boolean" defaultValue="false">
+        Defines whether or not aria-selected is added to the host element.

```

Comment:
1. Use monospace for attribute names.
2. "Defines" is redundant here.
3. "Whether" implies the possibility of "not."
```suggestion
        Whether <code>aria-selected</code> is added to the host element.

```